ports/stm/boards/stm32f746g_discovery/mpconfigboard.h

```diff
@@ -34,12 +34,15 @@
 #define FLASH_SIZE                  (0x100000)
 #define FLASH_PAGE_SIZE             (0x4000)
 
-#define BOARD_OSC_DIV (25)
-#define HSE_VALUE    ((uint32_t)25000000)
-#define BOARD_OSC_PLLN (400)
-#define BOARD_OSC_PLLQ (9)
+// Lower frequency to allow external RAM use
```

Comment:
Please explain this further. What clock needs to be lower to use external RAM? Why?

Reply:
@tannewt this is @k0d's deal, I don't know much about it. I just edited his values to match style.